MewsSystems/mews-flutter
# fix(optimus): [RND-103444] Fix visual issues in StepBar components

#### Summary
Fixed separator not changing color with completed steps. Issue mentions numbers not being aligned inside the step widget, but I don't see it in `master`. Could be some inconsistency with https://mews-optimus.web.app/#/stories/step-bar.

RND-103444
![image](https://user-images.githubusercontent.com/9210422/170515236-48e98d37-a916-4cbc-9ce1-091032212a79.png)

#### Testing steps

Open Storybook/Step Bar and fiddle with it. The separator's color should change, depending on the current step state.

#### Follow-up issues

None

#### Check during review

- Verify against YouTrack issue.
- Is the PR over 300 additions? Consider rejecting it with advice to split it. Is it over 500 additions? It should definitely be rejected.
- Unused code removed.
- Build passing.
- Is it a bug fix? Check that it is covered by a proper test (unit or integration).
